File: app/src/main/java/org/michaelbel/moviemade/rest/api/CREDITS.java
package org.michaelbel.moviemade.rest.api;

import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.Path;
import retrofit2.http.Query;

public interface CREDITS {

    @GET("credit/{credit_id}?")
    Observable<?> getDetails(
        @Path("credit_id") String id,
        @Query("api_key") String apiKey
    );
}

File: app/src/main/java/org/michaelbel/moviemade/data/service/CREDITS.kt
package org.michaelbel.moviemade.data.service

import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Path
import retrofit2.http.Query

interface CREDITS {

    @GET("credit/{credit_id}?")
    fun getDetails(
        @Path("credit_id") id: String,
        @Query("api_key") apiKey: String
    ): Observable<*>
}